docs/pages/development-guides/release-guide.md

```diff
@@ -1,4 +1,13 @@
-# How to Release
+---
+layout: default
+title: Release Guide
+parent: Development Guides
+nav_order: 3
+---
+
+# :tada: Release Guide
+
```

Comment:
Here

Reply:
So the release guide is not necessary or just as a comment?

Reply:
It is necessary, but instead of writing "You don't have to care about this guide" (your first comment) you can write "The Phonebook is automatically released. The steps below are executed each time you commit in the following format...".